### DISCUSSION TITLE
how run this program

### DISCUSSION THREAD

**Discussion by stly** [SCADA-LTS/Scada-LTS#2524] [Q&A] [answered]
@Limraj 
@grzesiekb 
do u run this program in idea IDE?
i deploy it but not work.
give me suggestion
thanks !

![image](https://user-images.githubusercontent.com/10585265/233818619-ded8d80f-c7db-4574-ba97-f82a8e9d857d.png)
![image](https://user-images.githubusercontent.com/10585265/233818621-c7d33c57-6d06-4e22-b5e7-86061901c00a.png)

**Comment by Limraj** [comment, marked as answer]
Hi @stly,
Look at tomcat and mysql configuration:
https://github.com/SCADA-LTS/Scada-LTS/wiki

Regards,
Kamil Jarmusik

**Reply by stly** [reply]
thanks a lot !

**Reply by stly** [reply]
@Limraj 
hi, you are a very nice man, thank your selfless answer！
i run programe according to the wiki, somthing wrong .
 " JDWP Transport dt_socket failed to initialize, TRANSPORT_INIT(510)  transport error 202: bind failed: Address already in use".
how can i kill the used address?

**Reply by Limraj** [reply]
Either you are trying to run tomcat a second time on the same port, or another process is using that port. I bet you didn't turn off tomcat and are trying to replay again. If you are working on windows, if intellij doesn't close processes properly for you, end them manually in Task Manager, all java processes. 

This is how you can stop a process in Intellij:
![Screenshot 2023-04-24 at 10 01 34](https://user-images.githubusercontent.com/35842300/233935391-3ec0b771-00b8-44d3-bf5b-647dc3062331.png)

If the process shuts down successfully, the 'Run' icon appears:
![Screenshot 2023-04-24 at 10 01 57](https://user-images.githubusercontent.com/35842300/233935468-668a7493-2078-48e5-b52a-3bbeae4aabda.png)

On unix systems (linux, macos > X) this should work fine, but for example on windows 7 it doesn't work so well anymore, you have to close these processes manually.

**Reply by stly** [reply]
thanks a lot @Limraj
